src/auth-users/services/create-user.service.ts

```
@@ -0,0 +1,51 @@
+import { PrismaClient } from "@prisma/client";
+import { SignUpEmailDto } from "../dto/sign-up/sign-up-email.dto";
+import { GetUserByEmail } from "./get-user-by-email.service";
+import { BadRequestException, ConflictException } from "@nestjs/common";
+import { UserMetaData } from "src/utils/types/user-metadata";
+import { hash } from "argon2";
+import { GenerateOtpSecret } from "src/utils/others/encrypt-secret";
```

ESLint

[error] Replace `"src/utils/others/encrypt-secret"` with `'src/utils/others/encrypt-secret'`

(prettier/prettier)